samples/test-samples.sh

```diff
@@ -29,7 +29,7 @@ if test "$1" = "-i" ; then
     BIN_DIR=$(realpath $2)
     INSTALL_DIR=$(realpath -m $4$3)
     rm -rf "$INSTALL_DIR"
-    make -C "$BIN_DIR" "DESTDIR=$4" install
+    cd "$BIN_DIR" && DESTDIR=$4 cmake --build . --target install
```

Comment:
Can you achieve the same thing without changing the directory? Would the following work?
cmake --build $BIN_DIR --target install